ci/prow/e2e-incluster-build

```diff
@@ -0,0 +1,70 @@
+##!/usr/bin/env bash
+
+set -euxo pipefail
+
+echo "Deploy KMMO..."
+make deploy
+
+echo "Wait until the KMMO Deployment is Available"
+echo "And Describe the Deployment / pods and get their YAML if that failed..."
+timeout 1m kubectl wait --for condition=Available deployments.apps -n kmm-operator-system kmm-operator-controller-manager || kubectl describe deployments.apps -n kmm-operator-system kmm-operator-controller-manager && kubectl get -o yaml deployments.apps -n kmm-operator-system kmm-operator-controller-manager && kubectl describe pod -n kmm-operator-system && kubectl get -o yaml pod -n kmm-operator-system
+
+echo "Create a build secret..."
+kubectl create secret generic build-secret --from-literal=ci-build-secret=super-secret-value
+
+      # The minikube registry-alias addon creates a Job that adds registry.minikube to the CoreDNS configuration.
+      # https://github.com/kubernetes/minikube/blob/master/deploy/addons/registry-aliases/patch-coredns-job.tmpl
+      # This job sometimes does not finish before the operator starts looking for the image in the registry, which
+      # results in failed DNS resolution.
+      # Add a job that tries to resolve registry.minikube, like the operator.
+      # Wait up to 6 minutes, which corresponds to our job's maximum lifetime including backoffs.
+echo "Wait for the internal registry to be available in CoreDNS"
+echo "and if fails check the status of components in the kube-system namespace..."
+kubectl apply -f ci/job-wait-minikube-registry-alias.yaml
+timeout 6m kubectl wait --for=condition=Complete --timeout -1s job/wait-minikube-registry-alias || kubectl get all -n kube-system
+
+echo "Add a configmap that contain the kernel module build dockerfile..."
+kubectl apply -f ci/kmm-kmod-dockerfile.yaml
+
+echo "Add an kmm-ci Module that contains a valid mapping..."
+sed -e "s/KVER_CHANGEME/$(uname -r)/g" ci/module-kmm-ci-build.template.yaml | tee module-kmm-ci.yaml
+kubectl apply -f module-kmm-ci.yaml
+
+echo "Wait for the job to be created..."
+timeout 1m bash -c 'until kubectl get job -l kmm.node.kubernetes.io/module.name | grep kmm; do sleep 1; done'
```

Comment:
```suggestion
echo "Add an kmm-ci Module that contains a valid mapping..."
sed -e "s/KVER_CHANGEME/$(uname -r)/g" ci/module-kmm-ci-build.template.yaml | tee module-kmm-ci.yaml
kubectl apply -f module-kmm-ci.yaml
timeout 1m bash -c 'until kubectl get job -l kmm.node.kubernetes.io/module.name | grep kmm; do sleep 1; done'
```

Reply:
 It already [exists](https://github.com/kubernetes-sigs/kernel-module-management/blob/main/.github/workflows/e2e.yaml#L377) in current tests and it looks relevant info in my opinion. Why removing that comment?

Reply:
Same reason as before, this is my option only and others may think other wise but eventually we are "applying a module", all the rest is just implementation details - so we are adding redundant info that shadows the main step which is just applying a module.

If you don't agree with this we can keep it.

Reply:
I would leave it. When reading the log file, the comment is easier to understand that the command.